### PR TITLE
Updated project with latest dependencies and Swift 5

### PR DESCRIPTION
**Pull request by SeanAsby**
Went to the issues section to see if there was an issue open for updating to Swift 5. I saw the reply that a pull request should be opened so I thought I'd give it a go. 

Please let me know if there is anything else that should be updated.

### CODE REVIEW

File: NotificationBanner/Classes/NotificationBannerQueue.swift [hunk 1]
```diff
@@ -49,7 +49,7 @@ open class NotificationBannerQueue: NSObject {
         if queuePosition == .back {
             banners.append(banner)
             
-            if banners.index(of: banner) == 0 {
+			if banners.firstIndex(of: banner) == 0 {
```
Comment (Daltron) [inline NotificationBanner/Classes/NotificationBannerQueue.swift]: This line has too much spacing

Comment (SeanAsby) [reply NotificationBanner/Classes/NotificationBannerQueue.swift]: This was due to Xcode being set to use tabs for indentation and GitHub rendering it incorrectly. Changed to spaces now.

**Comment by Daltron** [comment]
@SeanAsby Thanks for doing this! I'll have a new CocoaPods release shortly!

**Comment by weakfl** [comment]
@Daltron can you please push the latest version to cocoapods?

**Comment by weakfl** [comment]
@Daltron also, shouldn't the `MarqueeLabel` dependency be changed to [v4.0.0](https://github.com/cbpowell/MarqueeLabel/releases)?

```
s.dependency 'MarqueeLabel/Swift', '~> 4.0'
```

**Pull request merged**